custom/ewsghana/api.py

```diff
@@ -658,6 +658,9 @@ def web_user_sync(self, ews_webuser):
             if self.domain not in user.get_domains():
                 user.add_domain_membership(self.domain, location_id=location_id)
 
+            # We are migrating only active users
+            user.is_active = True
```

Comment:
I've just noticed that some of users are mark as inactive in HQ but active in EWS. I guess that they were previously marked as inactive in v1 and migrated over as inactive.